src/Razor/src/Microsoft.AspNetCore.Razor.LanguageServer/IOnInitialized.cs

```diff
@@ -3,11 +3,12 @@
 
 using System.Threading;
 using System.Threading.Tasks;
+using Microsoft.CommonLanguageServerProtocol.Framework;
 using Microsoft.VisualStudio.LanguageServer.Protocol;
 
 namespace Microsoft.AspNetCore.Razor.LanguageServer;
 
 internal interface IOnInitialized
 {
-    Task OnInitializedAsync(VSInternalClientCapabilities clientCapabilities, CancellationToken cancellationToken);
+    Task InitializeAsync(ILspServices services, CancellationToken cancellationToken);
```

Comment:
I have no issue with getting rid of the "On", but "initialize" and "initialized" are both LSP concepts, and I think it could be confusing not to be clear about this being past tense. They're essentially before and after server initialization.

Reply:
I wasn't thinking of it that way. In my mind, this is just a mechanism for services to be initialized. In that sense, the notion that service initialization is tied to LSP initialization is an implementation detail, and could potentially even change.

Reply:
I think concepts are being mixed here. I'm not sure all of this initialization is the same. For example, `IClientConnection`'s `IOnInitialized` implementation is intended to ensure that it doesn't make any requests until everything is initialized. However, it might now be initialized before other services that implement `IOnInitialized`, even though it really wants to be last.

I'm not sure that `IOnInitialized` is the "right" way to do this. It's feeling clunky to me.

Reply:
Happy to talk about this further, as I'm not sure I agree, or probably more correctly, fully understand your perspective. :)

Reply:
For sure! I ended up changing it to match your preference, so we can chat about it at your leisure.